python/cugraph/cugraph/gnn/feature_storage/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
possibly a minor thing, but I'm assuming this and all others in this PR should just be 2023?

Reply:
Yeah, I just copied it. Will make it 2023.

Reply:
Resolved by https://github.com/rapidsai/cugraph/pull/3139/commits/ae689fcd6609e3dcd946b7240c9e75ce6f5d7a1a